src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -393,6 +498,13 @@ export default {
       }
       return true;
     },
+    getValidationErrorIcon(errorField) {
+      const icons = Object.keys(VALIDATION_ICONS).filter((field) => field.includes(errorField));
```

Comment:
What's wrong with the naive `return VALIDATION_ICONS[errorField] || VALIDATION_ICONS.default`?

Also, what's the motivation for doing `field.includes(errorField)` instead of `errorField.includes(field)`? IIRC `errorField` might be something nested like `assetsSummary.creator.0.name`, but `field` will basically never be a superset of an `errorField`.

Reply:
> What's wrong with the naive `return VALIDATION_ICONS[errorField] || VALIDATION_ICONS.default`?

The intention was to support something like `assetsSummary.creator.0.name`, where the correct entry in `VALIDATION_ICONS` would be `name`.

> Also, what's the motivation for doing `field.includes(errorField)` instead of `errorField.includes(field)`? IIRC `errorField` might be something nested like `assetsSummary.creator.0.name`, but `field` will basically never be a superset of an `errorField`.

You're right, I swapped those around. https://github.com/dandi/dandiarchive/pull/796/commits/2ac2d1ef637b9e945caffbc942c1c5ec575f8220